org.eclipse.editorconfig.ui/src/main/java/org/eclipse/editorconfig/ui/internal/EditorConfigVisitor.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Nathan Jones, Jackson Bailey
```

Comment:
@ncjones I put your name here even though you haven't touched this file since all the code was copied and pasted from a file you wrote.